dashboard/src/shared/schema.ts

```diff
@@ -90,7 +92,8 @@ export function setValue(values: string, path: string, newValue: any) {
 export function getValue(values: string, path: string, defaultValue?: any) {
   const doc = YAML.parseDocument(values);
   const splittedPath = path.split(".");
-  return (doc as any).getIn(splittedPath) || defaultValue;
+  const value = (doc as any).getIn(splittedPath);
+  return value === undefined || value === null ? defaultValue : value;
```

Comment:
Not really important, but should the default also be limited to min/max? Completely ignorable :)

Reply:
the default is set by the user (the same way they set the min/max) so that shouldn't be a case (and their tests should catch that)